marimo/_plugins/ui/_impl/tables/narwhals_table.py

```diff
@@ -100,6 +101,11 @@ def select_rows(self, indices: list[int]) -> TableManager[Any]:
     def select_columns(self, columns: list[str]) -> TableManager[Any]:
         return self.with_new_data(self.data.select(columns))
 
+    def select_cells(self, cells: list[Cell]) -> TableManager[Any]:
+        df = self.as_frame()
+        # This does not work and I could use some help
+        return self.with_new_data([df.at[row, col] for row, col in cells])
```

Comment:
Hello @mscolnick, any ideas what I need to here?
I'm a bit confused what exactly should be returned here.
This code doesn't work because of narwhal stuff

![image](https://github.com/user-attachments/assets/d54ea26c-c8a4-492d-9799-7ea17b2c9a15)

But I'm also unclear if I need this.

Because I'm unsure how this would play out if the `on_change` callback?